js/ui/overlay/ui.overlay.js

```diff
@@ -932,16 +934,22 @@ const Overlay = Widget.inherit({
 
     _moveFromContainer: function() {
         this._$content.appendTo(this.$element());
-
-        this._detachWrapperToContainer();
+        this._$wrapper.detach();
     },
 
-    _detachWrapperToContainer: function() {
-        this._$wrapper.detach();
+    _checkContainerExists() {
+        const $wrapperContainer = this._positionController.$container;
+
+        const containerExists = $($wrapperContainer).length > 0;
```

Comment:
Do we need to wrap $($wrapperContainer) twice?

Reply:
Fixed. Thanks!